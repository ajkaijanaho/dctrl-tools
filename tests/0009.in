# abc
Foo: bar
# baz
Xyzzy: foo
#ytty

# ccd
Sdd: gh
goo: gaa
