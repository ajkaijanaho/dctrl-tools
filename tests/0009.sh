#!/bin/sh

set -e

$GREP_DCTRL ''
