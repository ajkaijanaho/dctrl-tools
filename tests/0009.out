Foo: bar
# baz
Xyzzy: foo
#ytty

Sdd: gh
goo: gaa

